pkg/utils/strings.go

```diff
@@ -63,3 +65,17 @@ func Substr(str string, start, length int) string {
 
 	return string(rs[start:end])
 }
+
+func Strings2Bytes(s string) (b []byte) {
```

Comment:
Maybe use `String2Bytes` function name

Reply:
Done. change function name to String2Bytes